### PR TITLE
Add more invoke examples

### PR DESCRIPTION
**Pull request by brilang**
I found myself looking for similar examples, so I figured I should add them.

### CODE REVIEW

File: data/en/invoke.json [hunk 1]
```diff
@@ -25,6 +25,24 @@
 			"description":"Invokes the size method on a new HashMap object, which should return 0",
 			"code":"invoke(createObject(\"java\", \"java.util.HashMap\"), \"size\")",
 			"result":"0"
+		},
+		{
+			"title":"Invoke a method on a component",
```
Comment (shaedrich) [inline data/en/invoke.json]: I would add "(script syntax)" to the second and "(tag syntax)" to the first example for clarity.

File: data/en/invoke.json [hunk 2]
```diff
@@ -25,6 +25,24 @@
...
+		{
+			"title":"Invoke a method on a webservice with one argument",
+			"description":"Invokes the method named 'test' on the webservice Test.cfc with one argument",
+			"code":"obj = createObject(\"webservice\", \"https://example.com/test.cfc?wsdl\");\rinvoke(obj, \"test\", {argument1=\"Test Data\"});",
+			"result":""
+		},
+		{
+			"title":"Invoke a method on a webservice with multiple arguments",
+			"description":"Invokes the method named 'test' on the webservice Test.cfc with multiple arguments",
+			"code":"obj = createObject(\"webservice\", \"https://example.com/test.cfc?wsdl\");\rinvoke(obj, \"test\", {argument1=\"Test Data\", argument2=\"More Data\", argument3=\"Still More Data\"});",
```
Comment (shaedrich) [inline data/en/invoke.json]: Is there any difference to invoking it with just one argument?

File: data/en/invoke.json [hunk 3]
```diff
@@ -25,6 +25,24 @@
 			"description":"Invokes the size method on a new HashMap object, which should return 0",
 			"code":"invoke(createObject(\"java\", \"java.util.HashMap\"), \"size\")",
 			"result":"0"
+		},
+		{
+			"title":"Invoke a method on a component",
+			"description":"Invokes the method named 'test' on the component Test.cfc with one parameter",
+			"code":"obj = createObject(\"component\", \"Test\");\rinvoke(obj, \"test\", {parameter=\"Test Data\"});",
```
Comment (KamasamaK) [inline data/en/invoke.json]: You normally wouldn't have `\r` alone. It would either be `\r\n`, or preferably just `\n`. Same applies below.

**Comment by pfreitag** [comment]
@brilang thanks for contributing! I will merge this and since you are a first time contributor, I'll go ahead and make the additional changes that @shaedrich and @KamasamaK suggested.

**Comment by brilang** [comment]
Somehow I missed this in my email. Thanks for making those changes @pfreitag. I'll take 
@shaedrich and @KamasamaK advice into account for future pull requests.

**Pull request merged**